tools/packaging/scripts/configure-hypervisor.sh

```diff
@@ -445,7 +454,9 @@ generate_qemu_options() {
 	_qemu_cflags=""
 
 	# compile with high level of optimisation
-	_qemu_cflags+=" -O3"
+	if [ "${qemu_version_major}" -le 4 ] || [[ "${qemu_version_major}" -eq 5 && "${qemu_version_minor}" -lt 2 ]]; then
```

Comment:
please use the same "format":  `[[ condition1 || (condition2 && condition3)  ]]`